docs/release/releases/release-1.12.md

```diff
@@ -0,0 +1,37 @@
+# Cluster API v1.12
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.12` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 18th August 2025     | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 22nd August 2025     | week 1   |
+| Team finalized                                       | Release Lead | Friday 22nd August 2025     | week 1   |
+| *v1.10.x & v1.11.x released*                         | Release Lead | Tuesday 9th September 2025  | week 4   |
```

Comment:
```suggestion
| *v1.10.x & v1.11.x released*                         | Release Lead | Tuesday wth September 2025  | week 4   |
```
According to https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/releases/release-1.11.md we have to release 1.11.1 the Nov 2nd (~1 week  after K8s 1.34 GA).
Considering this, I would do in the same day also the patch release for 1.10 so we are done for November